platformio.ini

```diff
@@ -161,7 +161,7 @@ src_filter        = ${common.default_src_filter} +<src/HAL/HAL_LPC1768>
 monitor_speed     = 250000
 lib_deps          = Servo
   LiquidCrystal
-  https://github.com/MarlinFirmware/U8glib-HAL/archive/dev.zip
+  U8glib-HAL=https://github.com/MarlinFirmware/U8glib-HAL/archive/dev.zip
```

Comment:
This is a useful syntax. May be helpful in other places too.

Reply:
I thought so too, I added it to all the downloaded zip libs to clean things up in the first commit, but DUE builds failed for errors with no apparent cause so reverted it.